src/firestore/firestore-internal.ts

```diff
@@ -16,13 +16,51 @@
 
 import { FirebaseApp } from '../firebase-app';
 import { FirebaseFirestoreError } from '../utils/error';
-import { FirebaseServiceInterface, FirebaseServiceInternalsInterface } from '../firebase-service';
 import { ServiceAccountCredential, isApplicationDefault } from '../auth/credential';
-import { Firestore, Settings } from '@google-cloud/firestore';
+import { Settings, Firestore } from '@google-cloud/firestore';
+import { FirebaseServiceInterface, FirebaseServiceInternalsInterface } from '../firebase-service';
 
 import * as validator from '../utils/validator';
 import * as utils from '../utils/index';
 
+export function getFirestoreOptions(app: FirebaseApp): Settings {
```

Comment:
Why does this file contain code changes?

Reply:
Sorry, it shouldn't have any changes. I had originally put some code in `firestore.ts`, but after realizing it wasn't necessary and adding it back in a different order polluted the diff. Fixed.